backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -155,6 +157,38 @@ public Project Get(int id)
             return project;
         }
 
+        /// <summary>
+        /// Get the project for the specified 'id'.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="includes"></param>
+        /// <returns></returns>
+        public Project Get(int id, params Expression<Func<Project, object>>[] includes)
```

Comment:
Provides us a basic way to only return what we need.  Need to figure out how to do a `ThenInclude` like this too.